azkaban-exec-server/src/main/java/azkaban/container/FlowContainer.java

```diff
@@ -378,6 +381,29 @@ private void createFlowRunner(final ExecutableFlow flow) throws ExecutorManagerE
       watcher = new RemoteFlowWatcher(pipelinedExecId, this.executorLoader);
     }
 
+    int numJobThreads = this.numJobThreadPerFlow;
```

Comment:
This code looks very similar to what we already have. Is it possible to refactor it?